perfdash/perfdash.go

```diff
@@ -193,7 +54,7 @@ func main() {
 		}
 	}()
 
-	http.Handle("/api", &buildLatency)
+	http.Handle("/result", &result)
```

Comment:
Please don't change the endpoint if you don't have a very, very, very good reason. I don't see one here.